src/helpers/contexts/GlobalNodeState.tsx

```diff
@@ -166,8 +166,24 @@ export const GlobalProvider = ({
     );
 
     const dumpState = useCallback((): SaveData => {
+        const nodesToSave = nodes.map((n) => ({
+            data: {
+                schemaId: n.data.schemaId,
+                inputData: n.data.inputData,
+                id: n.data.id,
+                ...(n.data.iteratorSize ? { iteratorSize: n.data.iteratorSize } : {}),
+            },
+            id: n.id,
+            position: n.position,
+            type: n.type,
+            selected: n.selected,
+            ...(n.height ? { height: n.height } : {}),
+            ...(n.width ? { width: n.width } : {}),
+            ...(n.zIndex ? { zIndex: n.zIndex } : {}),
```

Comment:
Why? `n.height` is `number | undefined`, right? This means that you can just say `height: n.height`. If height is undefined, JSON.stringify will remove the property: E.g.

```js
> JSON.stringify({ foo: undefined })
'{}'
```